scripts/export.js

```diff
@@ -0,0 +1,188 @@
+
+    const img = document.createElement('img');
+    img.className = 'speaker-img';
+    img.alt = each.image.alt;
+    img.src = each.image.url;
+
+    speakerImgDiv.appendChild(img);
+
+    const speakerInfoDiv = document.createElement('div');
+    speakerInfoDiv.className = 'speaker-info-div';
+
+    const h3 = document.createElement('h3');
+    h3.innerHTML = each.name;
+    speakerInfoDiv.appendChild(h3);
+
+    const span = document.createElement('span');
+    span.innerText = each.profession;
+    speakerInfoDiv.appendChild(span);
+
+    const hr = document.createElement('hr');
+    speakerInfoDiv.appendChild(hr);
+
+    const p = document.createElement('p');
+    p.innerText = each.bio;
+    speakerInfoDiv.appendChild(p);
+
+    speakerDiv.appendChild(speakerImgDiv);
+    speakerDiv.appendChild(speakerInfoDiv);
+
+    if (each.batch === 1) {
+      groupSpeakersDiv.appendChild(speakerDiv);
+    } else if (each.batch === 2) {
+      groupSpeakersDiv2.appendChild(speakerDiv);
+    } else {
+      groupSpeakersDiv3.appendChild(speakerDiv);
+    }
+  });
+
+  speakersDivContainer.appendChild(groupSpeakersDiv);
+
+  const collapseButtonDiv = document.createElement('div');
+  collapseButtonDiv.className = 'do-collapse-div';
+  collapseButtonDiv.id = 'do-collapse-div';
+  const collapseButton = document.createElement('button');
+  collapseButton.type = 'button';
+  collapseButton.className = 'more-button';
+  collapseButton.id = 'do-collapse';
+  collapseButton.innerText = 'MORE';
+
+  const buttonImg = document.createElement('img');
+  buttonImg.src = 'images/more-arrow.PNG';
+  buttonImg.alt = '';
+  collapseButton.addEventListener('click', showGuests);
+  collapseButton.appendChild(buttonImg);
+  collapseButtonDiv.appendChild(collapseButton);
+
+  speakersDivContainer.appendChild(collapseButtonDiv);
+
+  const collapseDiv = document.createElement('div');
+  collapseDiv.className = 'hide block';
+  collapseDiv.id = 'collapse';
+
+  collapseDiv.appendChild(groupSpeakersDiv2);
+  collapseDiv.appendChild(groupSpeakersDiv3);
+
+  speakersDivContainer.appendChild(collapseDiv);
+
+  const undoCollapseButtonDiv = document.createElement('div');
+  undoCollapseButtonDiv.className = 'hide undo-collapse-div';
+  undoCollapseButtonDiv.id = 'undo-collapse-div';
+
+  const undoCollapseButton = document.createElement('button');
+  undoCollapseButton.className = 'less-button';
+  undoCollapseButton.id = 'undo-collapse';
+  undoCollapseButton.type = 'button';
+  undoCollapseButton.addEventListener('click', hideGuests);
+
+  const buttonImage = document.createElement('img');
+  buttonImage.src = 'images/Less-arrow.png';
+  buttonImage.alt = '';
+  undoCollapseButton.appendChild(buttonImage);
+
+  undoCollapseButtonDiv.appendChild(undoCollapseButton);
+
+  speakersDivContainer.appendChild(undoCollapseButtonDiv);
+}
+
+const section = document.getElementById('');
+if (section !== null) {
+  section.addEventListener('click', loadSpeakers);
+}
```

Comment:
- [ ] Great implementation here to use Js file to keep the image and description for the featured speakers however, let us add some decoration to the image.

-----------------------------------------------Yours---------------------------------------

![cpp16](https://user-images.githubusercontent.com/55635977/132905452-2c063ed9-2ef4-42fc-8183-91dfe5d04a1f.png)

------------------------------------------Original---------------------------------------------

![cpp17](https://user-images.githubusercontent.com/55635977/132905457-637feea9-5c0b-49b8-954c-9328cb37a0e9.png)